### PR TITLE
chore: enable napi tests

### PR DESCRIPTION
**Pull request by williamluke4**

### CODE REVIEW

**Comment by Jolg42** [comment]
I think you can merge the dev branch in here

File: platforms-serverless/azure-functions-windows/run.sh [hunk 1]
```diff
@@ -15,9 +15,12 @@ mv "func-placeholder" "$app"
 group="prisma-e2e-windows"
 storage="prismae2ewin6owsstorage"
 
-az functionapp create --resource-group "$group" --consumption-plan-location westeurope --name "$app" --storage-account "$storage" --runtime "node" --os-type Windows
+az functionapp create --resource-group "$group" --consumption-plan-location westeurope --name "$app" --storage-account "$storage" --runtime "node" --os-type Windows --functions-version 3
+
 az functionapp config appsettings set --name "$app" --resource-group "$group" --settings "DEBUG=*"
-az functionapp config appsettings set --name "$app" --resource-group "$group" --settings "AZURE_FUNCTIONS_WINDOWS_PG_URL=$AZURE_FUNCTIONS_WINDOWS_PG_URL"
+az functionapp config appsettings set --name "$app" --resource-group "$group" --settings "AZURE_FUNCTIONS_LINUX_PG_URL=$AZURE_FUNCTIONS_LINUX_PG_URL"
```
Comment (janpio) [inline platforms-serverless/azure-functions-windows/run.sh]:
```suggestion
az functionapp config appsettings set --name "$app" --resource-group "$group" --settings "AZURE_FUNCTIONS_WINDOWS_PG_URL=$AZURE_FUNCTIONS_WINDOWS_PG_URL"
```

File: platforms-serverless/azure-functions-linux/finally.sh [hunk 2]
```diff
@@ -6,7 +6,12 @@ app="$(cat func-tmp.txt)"
...
 
-az functionapp delete --name "$app" --resource-group "$group"
+subscriptionId="63cbd994-4c44-4b0d-ba9f-463c6ec6ee31"
 
-az config set extension.use_dynamic_install=yes_without_prompt
-az monitor app-insights component delete --app "$app" --resource-group "$group"
+# az functionapp delete --name "$app" --resource-group "$group"
+
+# az config set extension.use_dynamic_install=yes_without_prompt
+# az monitor app-insights component delete --app "$app" --resource-group "$group"
```
Comment (janpio) [inline platforms-serverless/azure-functions-linux/finally.sh:14]: We need to undo this before merging.

File: platforms-serverless/azure-functions-linux/func-placeholder/index.ts [hunk 3]
```diff
@@ -1,9 +1,28 @@
...
-const client = new PrismaClient()
+import { Context, HttpRequest } from '@azure/functions'
+import { Prisma, PrismaClient } from '@prisma/client'
+const client = new PrismaClient({
+  datasources: {
+    db: {
+      url: process.env.AZURE_FUNCTIONS_LINUX_PG_URL as string,
+    },
+  },
+})
```
Comment (janpio) [inline platforms-serverless/azure-functions-linux/func-placeholder/index.ts:9]: This should not really work.

File: platforms-serverless/azure-functions-linux/prisma/schema.prisma [hunk 4]
```diff
@@ -1,11 +1,12 @@
 generator client {
   provider      = "prisma-client-js"
   binaryTargets = ["native", "debian-openssl-1.1.x"]
+  previewFeatures = ["napi"]
 }
 
 datasource db {
   provider = "postgresql"
-  url      = env("AZURE_FUNCTIONS_LINUX_PG_URL")
+  url      = "postgresql://postgres:posgres@localhost:5432/postgres=?schema=public"
```
Comment (janpio) [inline platforms-serverless/azure-functions-linux/prisma/schema.prisma]: needs to be undone

File: platforms-serverless/azure-functions-linux/prisma/schema.prisma [hunk 5]
```diff
@@ -1,11 +1,12 @@
 generator client {
   provider      = "prisma-client-js"
   binaryTargets = ["native", "debian-openssl-1.1.x"]
+  previewFeatures = ["napi"]
```
Comment (janpio) [inline platforms-serverless/azure-functions-linux/prisma/schema.prisma]: should not be necessary

Comment (williamluke4) [reply platforms-serverless/azure-functions-linux/prisma/schema.prisma]: removed

File: platforms-serverless/azure-functions-linux/prisma/schema.prisma (hunk 4 above)
Comment (williamluke4) [reply platforms-serverless/azure-functions-linux/prisma/schema.prisma]: reverted

File: platforms-serverless/azure-functions-linux/func-placeholder/index.ts (hunk 3 above)
Comment (williamluke4) [reply platforms-serverless/azure-functions-linux/func-placeholder/index.ts:9]: Do you mean that this should not be nessasary?

**Comment by williamluke4** [comment]
Closing as this has been implement here #1717

**Pull request closed without merging**